life_expectancy/tests/fixtures/pt_life_expectancy_expected.csv

```diff
@@ -1182,14 +1182,7 @@ YR,M,Y6,PT,2016,72.5
 YR,M,Y60,PT,2016,21.8
 YR,M,Y61,PT,2016,21.0
 YR,M,Y62,PT,2016,20.3
-YR,M,Y63,PT,2016,19.5
-YR,M,Y64,PT,2016,18.8
-YR,M,Y65,PT,2016,18.0
-YR,M,Y66,PT,2016,17.2
-YR,M,Y67,PT,2016,16.5
-YR,M,Y68,PT,2016,15.8
-YR,M,Y69,PT,2016,15.1
-YR,M,Y7,PT,2016,71.5
+YR,M,Y63,
```

Comment:
Why are you expecting a row with neither region nor year in it?

Reply:
I believe this table was already in the base repo 👀